Vizzuality/widget-editor
# Fix the format of the axes and values in  the tooltip

This PR fixes formatting issues of values in the axes and tooltip. Particularly, temporal axis are now displayed as in v1.

## Testing instructions

There are a lot of cases to test (per chart type). Here's one example of widget that now correctly displays the axes and values in the tooltip: `bcbdf5db-6e27-4250-9b74-ce238ef2bde9`.

You should see the ticks of the X axis displaying years correctly. Feel free to change to a different chart type or to update the format of the Y axis to see changes in the axis and the tooltip.

Note the tooltip shows more decimals than the axis because the `nice` Vega setting is set for the axis (it rounds the values).

## Pivotal Tracker

- [Task 1](https://www.pivotaltracker.com/story/show/173175804)
- [Task 2](https://www.pivotaltracker.com/story/show/172821326)
